2024-11-25 16:51:12,267 - ERROR - Task was destroyed but it is pending!
task: <Task pending name='Task-156' coro=<Page.go_back() done, defined at D:\DATN\.venv\lib\site-packages\playwright\_impl\_page.py:586> wait_for=<Future pending cb=[Task.task_wakeup()]> cb=[SyncBase._sync.<locals>.<lambda>() at D:\DATN\.venv\lib\site-packages\playwright\_impl\_sync_base.py:111, ProtocolCallback.__init__.<locals>.cb() at D:\DATN\.venv\lib\site-packages\playwright\_impl\_connection.py:195]>
